### DISCUSSION TITLE
如何关闭alist

### DISCUSSION THREAD

**Discussion by wuzhan666** [AlistGo/alist#3694] [Q&A]
使用alist stop 弹出 failed to find process by pid: 7584, reason: <nil>
进任务管理器中发现pid与上面的pid不一样

求助如何关闭alist

**Comment by youtehub** [comment]
什么操作系统，window系统 还是 linux

window 操作系统关闭指令
```
taskkill /f /im alist.exe
```
[stop_alist_v3.txt](https://github.com/alist-org/alist/files/10880520/stop_alist_v3.txt)


linux 系统操作指令 
```
  kill -9 `lsof -i:5244 | awk 'NR==2{print $2}'`     
```  
 5244替换成你的alist端口

**Reply by wuzhan666** [reply]
非常感谢

**Reply by ACEKaffy** [reply]
感谢！
